internal/pkg/cli/env_init_test.go

```diff
@@ -412,15 +433,42 @@ func TestInitEnvOpts_Ask(t *testing.T) {
 			inAppName: mockApp,
 			inEnv:     mockEnv,
 			inDefault: true,
+			getMockCredsSelector: func(c credsSelector) func() (credsSelector, error) {
+				return func() (credsSelector, error) {
+					return c, nil
+				}
+			},
```

Comment:
Can we set default value for this and override this behavior in just one of the test so that we don't need to repeat ourselves?

Reply:
I hope so, I'll take some more time looking into this but I'm not really sure how to without introducing some other changed behavior to each test (like using a struct constructor).